src/status_im/communities/core.cljs

```diff
@@ -175,11 +185,38 @@
   {:json-rpc/call [{:method      "wakuext_requestToJoinCommunity"
                     :params      [{:communityId community-id}]
                     :js-response true
-                    :on-success  #(re-frame/dispatch [::requested-to-join %])
+                    :on-success  #(do
```

Comment:
I have this one saved in my favorites, unfortunately the formatter can't automate for us.

By the Clojure Style Guide powers, I invoke `No multiple forms in function literals`! https://guide.clojure.style/#no-multiple-forms-fn-literals